### PR TITLE
Add ability to specify the quality of the image.

### PR DESCRIPTION
**Pull request by SteveNewson**
I've added the ability to specify the image quality when resizing an image. Added a couple of tests and updated the documentation.

### CODE REVIEW

File: lib/resizers/resizer_stream.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ ResizerStream.prototype.baseParameters = function() {
     output = this.options.convertTo + ':-';
   }
 
-  return ['-quality', '91', '+profile', '*', '-auto-orient', '-strip', output];
+  return ['-quality', this.options.quality ? this.options.quality : '91', '+profile', '*', '-auto-orient', '-strip', output];
```
Comment (paolochiodi) [inline lib/resizers/resizer_stream.js]: I think `this.options.quality || '91'` would be more concise. Wha do you think?

Comment (paolochiodi) [inline lib/resizers/resizer_stream.js]: And maybe it's a good idea to put the default quality in a variable... something like 
`var DEFAULT_QUALITY='91'`

**Comment by paolochiodi** [comment]
Thank you for the PR.
I've added a couple of comments.

I'm also interested to hear how are you using this and how did you find it.

**Comment by SteveNewson** [comment]
I've added the changes you suggested.

I've needed a CDN delivered image resizer and stumbled across this:

https://github.com/blomming/image-server

Which in turn uses your code to resize/crop the images. However, a quality of 91 wasn't good enough for what I needed (loads of ghosting on the images when they are reduced in size significantly). 

I've made my own copy of that server and hacked out all of the stuff I don't want or don't need.

**Comment by SteveNewson** [comment]
Not sure why the travis build is failing - works fine locally for me.

**Comment by paolochiodi** [comment]
No worries on the Trevis failure, I was never able to get graphics magick installed on it.
It's interesting about the image-server, I was a developer in blomming, helped building it and was the principal maintainer.
This lib was extracted from that server

**Comment by paolochiodi** [comment]
However, thank you for the changes.
I'll merge the PR and publish a new version later today or in the week end.

**Comment by paolochiodi** [comment]
Hi, I just published a new version with this PR. Let me know how this works.

**Pull request merged**